### PR TITLE
Layered edits to the home page.

### PR DESCRIPTION
**Pull request by akcano**
Renaming a branch closes the PRs associated with it, so I'm fully switching to the fork-based PR workflow this time.

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -3,67 +3,57 @@
...
-packages. Most such projects can organise setup complexity with Workspace.
-Examples include AI/ML, Robotics, IoT, EdTech and similar domains.
+=========
+
+**Workspace automates configuration provisioning
+in reproducible development environments**.
+
+**Define your dev environment in straightforward YAML**.
+The tool consumes the definition to create a contained workspace,
+installs the functionality it lists as a number of SDKs,
```
Comment (dmitry-lyfar) [inline docs/index.rst]: functionality => dependencies?

File: docs/index.rst [hunk 2]
```diff
@@ -3,67 +3,57 @@
...
-container.
-
-It is common to have a non-trivial project setup with dependencies on particular
-Linux distributions, SDKs from multiple publishers, and system and language
-packages. Most such projects can organise setup complexity with Workspace.
-Examples include AI/ML, Robotics, IoT, EdTech and similar domains.
+=========
+
+**Workspace automates configuration provisioning
+in reproducible development environments**.
```
Comment (dmitry-lyfar) [inline docs/index.rst]: Hmm, I doubt that "automates configuration" is the close enough term for whatever the users do when launch a dev environment. "Configuration provisioning" sounds closer to automating configuration for the DevOps tasks (e.g. see Configuration Management Tools, IaC). 

In our case, it is rather "Setting up (launching) the development environment" or "Install prerequisites" (I quickly googled and checked how some big OSS projects call it in the section of their docs for developers, see [comma.ai](https://github.com/commaai/openpilot/blob/master/docs/CONTRIBUTING.md), [gitlab prof services](https://about.gitlab.com/handbook/customer-success/professional-services-engineering/development-environment/), [supercollider](https://github.com/supercollider/supercollider/wiki/Setting-up-your-development-environment))?

File: docs/index.rst (hunk 1 above)
Comment (akcano) [reply docs/index.rst]: Yes, good one.

File: docs/index.rst (hunk 2 above)
Comment (akcano) [reply docs/index.rst]: @dmitry-lyfar "Workspace automates prerequisite setup in reproducible development environments", maybe?

Comment (dmitry-lyfar) [reply docs/index.rst]: @akcano that's closer. Should we drop the "reproducible development environments" as this is the solution? Given we want to speak about the problem it solves and, perhaps, not squeeze the solution into the same statement, something like: "Workspace automates development environment setup" or "Workspace automates intricate prerequisite setup for your project" (for the sake of the semantics wanted; not insisting on any statement). WDYT?

Comment (akcano) [reply docs/index.rst]: @dmitry-lyfar Let's see. We're following the standard home page template, and here's what it says about this part:
```
A single sentence that says what the product is, succinctly and memorably 
```
<https://docs.google.com/document/d/1Zw-UoQzHMSQjATLXjjvd9GKu5qU2mHmqxtKck8nGkbU/>

I'd like to put the accent on *what it is* as opposed to *what it does*, so, going with the latter of your suggestions:

```
Workspace is a tool that automates intricate prerequisite setup for your projects.
```

Anticipating a possible objection: I wouldn't go into much detail about the specific kind of tool it is (CLI, daemon, etc.), using the word *tool* in its broadest sense. A suitable synonym is possible, though.

Comment (dmitry-lyfar) [reply docs/index.rst]: Yes, I agree with the accent. The "tool" looks good there as well.

**Review by dmitry-lyfar — APPROVED** [review]

**Pull request merged**